ports/ceferino/Don Ceferino Hazaña.sh

```diff
@@ -31,13 +30,15 @@ export SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig"
 export XDG_DATA_HOME="$CONFDIR"
 export TEXTINPUTINTERACTIVE="Y" 
 
+# Must wait until bind_files comes out of beta
 $ESUDO rm -rf ~/.ceferino
 ln -sfv $GAMEDIR/conf/.ceferino ~/
+# bind_files ~/.ceferino $GAMEDIR/conf/.ceferino
```

Comment:
Requires new stable release before merge.

Reply:
Since `bind_files` is now in the stable release, these changes are also ready for release.